app/src/main/java/com/github/se/polyfit/viewmodel/map/MapViewModel.kt

```diff
@@ -0,0 +1,47 @@
+package com.github.se.polyfit.viewmodel.map
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.github.se.polyfit.data.remote.firebase.PostFirebaseRepository
+import com.github.se.polyfit.model.post.Location
+import com.github.se.polyfit.model.post.Post
+
+class MapViewModel(private val repository: PostFirebaseRepository = PostFirebaseRepository()) :
+    ViewModel() {
+
+  private val _location = MutableLiveData<Location>()
+  val location: LiveData<Location> = _location
```

Comment:
I could not find this anywhere; it might be a good idea to delete this if it is not used anywhere.

Reply:
it's going to be the "getter" for the map ui